index.js

```diff
@@ -311,6 +310,7 @@ function loader (registryOrVersion) {
     set durabilityUsed (value) {
       const where = registry.supportFeature('whereDurabilityIsSerialized')
       if (where === 'Damage') {
+        if (!value || value === 0) return
```

Comment:
Is the second condition not caught by the first? If it's not clear the first condition you can add as a comment

Reply:
Thought so but tests don't work without the second check for some reason

Reply:
That doesn't sound right, can you revert this so it fails?

Reply:
Doesn't fail without now. I probably made some other changes that caused the failures earlier